src/main/java/org/broadinstitute/dsm/model/birch/TestBostonResult.java

```diff
@@ -0,0 +1,14 @@
+package org.broadinstitute.dsm.model.birch;
+
+import lombok.Data;
+
+@Data
+public class TestBostonResult {
+    private String orderMessageId;
+    private String institutionId;
+    private String sampleId;
+    private String result;
+    private String reason;
+    private String timeCompleted;
+    
```

Comment:
How are we thinking of handling potential duplicates? We can do de-dup in Pepper, but we'll need to get the `is_corrected` in here as well.